spec/features/idv/previous_address_spec.rb

```diff
@@ -12,14 +12,6 @@ def expect_to_stay_on_verify_session_page
     expect(page).to have_selector("input[value='#{bad_zipcode}']")
   end
 
-  def expect_bad_previous_address_to_fail
-    fill_out_idv_form_ok
-    fill_out_idv_previous_address_fail
-    click_idv_continue
-
-    expect_to_stay_on_verify_session_page
-  end
-
```

Comment:
Why remove the scenario with the bad previous address?

Reply:
This was really testing whether one of the mock proofers to failed when given a bad zipcode in the previous address, its no different than the bad current address. I took it out because right now we don't have the concept of 'optional' attributes for the vendors since we're not doing anything with previous address.